docker/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN mkdir -p /opt/gcp_variant_transforms/bin && mkdir -p /opt/gcp_variant_transf
 ADD / /opt/gcp_variant_transforms/src/
 
 # Install dependencies.
+RUN apt install -y python3-pip
```

Comment:
Hmm, I am not sure if we need/should install pip with python3 support. Since it's a c++ issue, could you please try if updating gcc/g++ fixes it (`apt install g++`)?

Reply:
Installing g++ also fixes the problem.
Thanks!

Reply:
Great, could you also comment this is needed for mmh3 install?

Reply:
done.